instagram/src/Components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import  CommentSection  from '../CommentSection/CommentSection.js';
+import './PostContainer.css';
+
+export const PostContainer = props => {
+    function click(e) {
+        e.target.classList.toggle('PostContainer__icons-liked')
+        props.click(e.target.value);
+    }
+
+    function commentSubmit(e) {
```

Comment:
Take another look at this function - there is a bug here! 🐛

Reply:
Specifically at props.comment(e). What do you think that `props.comment(e)` is doing??